docs/plugins/verifier/notation.md

```diff
@@ -260,3 +260,19 @@ Sample Notation CLI config with timestamping configuration:
     }
 }
 ```
+
```

Comment:
Suggestion to add a table of content to page

Reply:
As there is a default ToC, should we mark this as  resolved https://github.com/ratify-project/ratify-web/pull/132#issuecomment-2560420009

Reply:
The default value for cache is recommended to true. The reason is:

- The current revocation checking procedue for Notary Project is to check OCSP first, and then fall back to CRL, so with OCSP, users can get the latest revocation status in time. This elimate the problem of the potential delay of CRL reovcation checking.
- The default to false may lead to CRL server throttling as each signature validation will trigger a CRL server connection

/cc @susanshi